### PR TITLE
OEL-253: Search default content.

### PR DESCRIPTION
**Pull request by timpevi**

### CODE REVIEW

**Comment by timpevi** [comment]
446 must be merged first.

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        "php": ">=7.3",
+        "drupal/core": "^8.9 || ^9.1",
+        "drupal/search_api": "^1.20"
+    },
+    "require-dev": {
+        "composer/installers": "~1.11",
+        "drupal/config_devel": "^1.2",
+        "drupal/core-composer-scaffold": "^8.9 || ^9.1",
+        "drupal/default_content": "^2.0@alpha",
+        "drupal/devel": "^4.1",
```
Comment (GilNovacomm) [inline composer.json]: Remove development modules such as Devel.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      # Enable step debugging for all PHP request. See ./README.md#step-debugging for more information.
+      # XDEBUG_SESSION: 1
+  mysql:
+    image: percona/percona-server:5.7
+    command: --innodb-log-file-size=1G --max_allowed_packet=1G --innodb-buffer-pool-size=512M --wait_timeout=3000 --net_write_timeout=3000 --log_error_verbosity=3
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
+    # ports:
+    #   - 3306:3306
+  phpmyadmin:
```
Comment (GilNovacomm) [inline docker-compose.yml]: Remove the PHPMyAdmin container it is a development aid.

File: modules/oe_showcase_search/README.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# OE Showcase Search
+Demo search feature based on search_api
```
Comment (GilNovacomm) [inline modules/oe_showcase_search/README.md]: This needs better description maybe:
Provides a sample content type oe_showcase_search_demo and content to show case the search features.

File: modules/oe_showcase_search/oe_showcase_search.info.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+name: OpenEuropa Library Search
+type: module
+description: 'Demo search feature based on search_api.'
```
Comment (GilNovacomm) [inline modules/oe_showcase_search/oe_showcase_search.info.yml:3]: Consider rewording in tandem with README description.

**Review by GilNovacomm — COMMENTED** [review]
Remove development aids such as PHPMyAdmin and Devel.

File: runner.yml.dist [hunk 5]
```diff
@@ -23,6 +23,9 @@ drupal:
     - "./vendor/bin/drush -y config-set system.performance js.preprocess 0"
     # Enable the modules.
     - "./vendor/bin/drush en oe_showcase_default_content -y"
+    - "./vendor/bin/drush en oe_showcase_search -y"
+    - "./vendor/bin/drush en search_api_db -y"
```
Comment (GilNovacomm) [inline runner.yml.dist]: Remove module install for search_api_db and search_api_db_defaults this should be required on oe_showcase_search

File: modules/oe_showcase_search/oe_showcase_search.info.yml [hunk 6]
```diff
@@ -6,7 +6,7 @@ package: OpenEuropa
 dependencies:
   - search_api:search_api
```
Comment (GilNovacomm) [inline modules/oe_showcase_search/oe_showcase_search.info.yml:7]: Add dependency on search_api_db_defaults

**Review by GilNovacomm — COMMENTED** [review]
Implementation approved pending Drone review.

**Review by GilNovacomm — DISMISSED** [review]

**Review by drishu — APPROVED** [review]

**Review by Maxfire — APPROVED** [review]
Reviewed, looks good.

**Review by GilNovacomm — APPROVED** [review]

**Pull request merged**